bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -189,7 +189,7 @@ public void channelUnlinked(ChannelUID channelUID) {
     protected void validateConfigurationParameters(Map<String, Object> configurationParameters)
             throws ConfigValidationException {
         ThingType thingType = TypeResolver.resolve(getThing().getThingTypeUID());
-        if (thingType != null) {
+        if (thingType != null && thingType.getConfigDescriptionURI() != null) {
```

Comment:
I do not dare to remove it right away, but I have entered https://github.com/eclipse/smarthome/issues/1062 for it.
Nonetheless, I prefer to leave this PR as it is - after all, I want to make sure that the return value of get is not null before I am using it, so this is what the code here does.